doc/release-notes/11172-fix-search-api-nested-metadata-fields.md

```diff
@@ -0,0 +1 @@
+Deeply nested compound fields are not (yet) supported by Dataverse but the Search API now properly avoids returning duplicate values for them. See #11172.
```

Comment:
@vera I added this. Please feel free to edit what I wrote!

Reply:
Thanks, looks good! :)